### PR TITLE
Feat/breadcrumbs

### PR DESCRIPTION
**Pull request by mpisacic**
Breadcrumbs component redesigned with changed props and behaviour.

### CODE REVIEW

File: src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,185 +1,138 @@
...
+        const paths = this.state.currentPath.map((item, index) => {
+                const iconName = item.selected ? 'icon-arrow_down_right' : 'icon-arrow_right';
+                return (
+                    <li className={'breadcrumbs-list-item'} key={index} >
+                        <Dropdown
+                            ref={this.setDropdownReference}
+                            dropdownType={DropdownType.customDropdown} 
+                            icon={iconName} 
+                            onClosed={() => this.closeMenu(item)} 
+                            onOpenedClick={(willOpen) => this.openMenu(item, willOpen)}
```
Comment (hbagarich) [inline src/components/Breadcrumbs/Breadcrumbs.tsx]: Do the breadcrumb component needs to track this?

File: src/components/Dropdown/Dropdown.Props.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ export interface IDropdownProps {
   calloutClassName ?: string;
   layerClassName ?: string;
   onClosed?: () => void;
+  onOpenedClick?: (opened?: boolean) => void;
```
Comment (hbagarich) [inline src/components/Dropdown/Dropdown.Props.ts]: Is this event needed?

File: src/components/Breadcrumbs/Breadcrumbs.tsx (hunk 1 above)
Comment (mpisacic) [inline src/components/Breadcrumbs/Breadcrumbs.tsx]: Yes, for displaying menu.

File: src/components/Dropdown/Dropdown.Props.ts (hunk 2 above)
Comment (mpisacic) [inline src/components/Dropdown/Dropdown.Props.ts]: It is needed by breadcrumbs component, maybe in some other use cases.

**Pull request merged**